apollo-router/Cargo.toml

```diff
@@ -145,7 +145,7 @@ prost-types = "0.11.5"
 proteus = "0.5.0"
 rand = "0.8.5"
 rhai = { version = "1.12.0", features = ["sync", "serde", "internals"] }
-redis = { version = "0.21.7", optional = true, features = ["cluster", "tokio-comp"] }
+redis = { version = "0.21.7", optional = true, features = ["tokio-comp", "tls", "tokio-native-tls-comp"] }
```

Comment:
Does removing "cluster" not disable cluster support?

Reply:
the `cluster` feature of the redis crate gives access to the syncrhonous cluster client, while we use the asynchronous client from redis_cluster_async